internal/report.go

```diff
@@ -91,9 +92,134 @@ func GenerateReport(driverName string, conv *Conv, w *bufio.Writer, badWrites ma
 	if printUnexpecteds {
 		writeUnexpectedConditions(driverName, conv, w)
 	}
+
+	if conv.Audit.StreamingStats.Streaming {
+		writeStreamingMigrationReport(driverName, conv, w)
+	}
 	return summary
 }
 
+func sumNestedMapValues(mp map[string]map[string]int64) int64 {
```

Comment:
can be moved to utility

Reply:
If we move it to utility then there are import cycle issues. That's why this was implemented here.